### PR TITLE
docs: Add Ice and Rain Protection page

### PR DESCRIPTION
**Pull request by SweatyCam**
## Summary
Add page with information on the ice and rain protection systems

### Location
https://docs-git-fork-sweatycam-ice-and-rain-protection-flybywire.vercel.app/pilots-corner/advanced-guides/ice-rain-protection/

Discord username (if different from GitHub): Sweaty Cam#3177

### CODE REVIEW

File: docs/pilots-corner/advanced-guides/ice-rain-protection.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+# Ice and Rain Protection
+
+The A320 is exposed to many types of weather and atmospheric conditions, both in flight and on the ground. These include icing conditions and heavy rain, both of which could be potentially dangerous to the aircraft if they are not managed appropriately.
+
+!!! tip "Icing Conditions"
+    Icing conditions on ground and during take-off can be expected when the OAT (Outside Air Temperature) is at or below 10ᵒC and there exists visible moisture on the taxiways and runways such as standing water, slush, ice, or snow which may be ingested.    
+    Icing conditions in the air can be expected when the TAT (Total Air Temperature) is at or below 10ᵒC and the aircraft is flying through visible moisture such as cloud, fog with visibility of one mile or less, rain, snow, sleet, or ice crystals.
```
Comment (frankkopp) [inline docs/pilots-corner/advanced-guides/ice-rain-protection.md]:
```suggestion
!!! note "Definition of Icing Conditions"
    - Icing conditions exist when the OAT (on ground or after takeoff) or the TAT (in flight) is at or below 10°C 
      and visible moisture in any form is present (such as clouds, fog with visibility of 1 sm (1 600 m) or less, rain, 
      snow, sleet or ice crystals).
    - Icing conditions also exist when the OAT on the ground and for takeoff is at or below 10°C and operating on 
      ramps, taxiways or runways where surface snow, standing water or slush may be ingested by the engines, or 
      freeze on engines, nacelles or engine sensor probes.
```

Comment (frankkopp) [reply docs/pilots-corner/advanced-guides/ice-rain-protection.md]: Aligned it with the passage I go from one of our engineers

File: docs/pilots-corner/advanced-guides/ice-rain-protection.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+Jump to [Usage of the Ice and Rain Protection System](#usage-of-the-ice-and-rain-protection-system)
+
+## Wing Anti-Ice
+In flight, hot air from the pneumatic system heats the three outboard slats (3-4-5) of each wing.  
+Air is supplied through one valve in each wing.  
+The WING pushbutton on the ANTI-ICE panel controls the valves.  
+When the aircraft is on ground, the flight crew can initiate a 30-second test sequence by turning the system ON.  
+If the system detects a leak during normal operation, the affected side’s wind anti-ice valve automatically closes.  
+When wing anti-ice is selected, the N1 or EPR limit is automatically reduced, and the idle N1 or EPR limit is automatically increased.  
+If the electrical power supply fails, the valves close.
```
Comment (frankkopp) [inline docs/pilots-corner/advanced-guides/ice-rain-protection.md]:
```suggestion
## Wing Anti-Ice

- In flight, hot air from the pneumatic system heats the three outboard slats (3-4-5) of each wing.  
- Air is supplied through one valve in each wing.  
- The WING pushbutton on the ANTI-ICE panel controls the valves.  
- When the aircraft is on ground, the flight crew can initiate a 30-second test sequence by turning the system ON.  
- If the system detects a leak during normal operation, the affected side’s wind anti-ice valve automatically closes.  
- When wing anti-ice is selected, the N1 or EPR limit is automatically reduced, and the idle N1 or EPR limit is automatically increased.  
- If the electrical power supply fails, the valves close.
```

Might be more readable with bullet points. 
Same for the other paragraphs.

File: docs/pilots-corner/advanced-guides/ice-rain-protection.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+- Total Air Temperature (TAT) probes  
+
+Three independent Probe Heating Computers (PHCs) automatically control and monitor:  
+
+- Captain probes   
+- F/O probes  
+- STBY probes  
+
+They protect against overheating and indicate faults.  
+The probes are heated:
```
Comment (frankkopp) [inline docs/pilots-corner/advanced-guides/ice-rain-protection.md]:
```suggestion

The probes are heated:  
```

File: docs/pilots-corner/advanced-guides/ice-rain-protection.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+On the ground, the TAT probes are not heated and pitot heating operates at a low level. The changeover to normal power is automatic.
+
+## Rain Removal
+
+### Wipers
+
+Each front windshield has a two-speed electric wiper.
+A rotary selector controls each wiper.
+Wipers must never be used on a dry windshield.
```
Comment (frankkopp) [inline docs/pilots-corner/advanced-guides/ice-rain-protection.md]: Also maybe bullet points again.

**Review by frankkopp — DISMISSED** [review]
In general the content is good. I recommend using bullet points for most of the descriptions.

**Comment by Valastiri** [comment]
Will be reviewing this today. Sorry for the delay.

**Comment by Valastiri** [comment]
Suggest in `/pilots-corner/a32nx-briefing/flight-deck/ovhd/anti-ice/` add a button linking to your guide. See example: 
image

Otherwise lgtm. There is consideration to put your information into the briefing section instead as well.

**Comment by SweatyCam** [comment]
> Otherwise lgtm. There is consideration to put your information into the briefing section instead as well.

yeah, we could split the guide in two and move the top half of the page, the FCOM style info, into A32NX briefing, leaving just the “Usage of the ice and rain protection system” in the advanced guide. Your guys’ call

File: docs/pilots-corner/advanced-guides/ice-rain-protection.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+- Windshield heating operates at low power on the ground and at normal power in flight. The changeover is automatic.
+- Only one heating level exists for the windows.
+
+## Probes Heat  
+
+- Electrical heating protects:  
+
+      - Pitot heads  
+      - Static ports  
+      - Angle Of Attack (AOA) probes
```
Comment (Valastiri) [inline docs/pilots-corner/advanced-guides/ice-rain-protection.md]:
```suggestion
      - Angle of Attack (AOA) probes  
```

**Review by Valastiri — COMMENTED** [review]
small typo

**Comment by Valastiri** [comment]
check build

**Pull request merged**